flask_sqlalchemy/cli.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+"""
+    flask_sqlalchemy.cli
+    ~~~~~~~~~~~~~~~~~~~~
+
+    Command Line Interface for managing the database.
+
+    :copyright: (c) 2017 by David Baumgold.
+    :license: MIT, see LICENSE for more details.
+"""
+
+from __future__ import absolute_import
+
+from functools import wraps
+
+import click
+from flask import current_app
+from werkzeug.local import LocalProxy
+
+try:
+    from flask.cli import with_appcontext
+except ImportError:
+    from flask_cli import with_appcontext
```

Comment:
[Flask-CLI](https://pythonhosted.org/Flask-CLI/) supports Flask 0.10, and CLI support was added to Flask in 0.11. Flask 0.10 was released in June 2013, while Flask 0.11 was released in June 2016. Three years is hardly ancient history, even in software development.

Besides, the cost of support Flask-CLI is so low. Is it really a problem to keep this in?